src/node.cc

```diff
@@ -4250,6 +4245,13 @@ void Init(int* argc,
     exit(9);
   }
 
+  // CHAKRA-TODO : fix this to not do it here
```

Comment:
No problem, but I would like to find a way to not make any NODE_ENGINE_CHAKRACORE specific change to node.cc and somehow determine this in shim itself